tools/sof-kernel-log-check.sh

```
@@ -19,6 +18,12 @@ ignore_str="$ignore_str"'|error: status'
 # TODO explain
 ignore_str="$ignore_str"'|error: debugfs write failed to idle -16'
 
+# Realtek codecs thrown an error on startup, checking with Realtek
+ignore_str="$ignore_str"'|Parity error detected'
+
+# CML Helios known issue related with xhci_hcd
+ignore_str="$ignore_str"'|xhci_hcd 0000:00:14.0: WARN Set TR Deq Ptr cmd failed due to incorrect slot or ep state'
```

Please one context URL (bug, mailing-list discussion, anything...) per ignored error. File a new bug/task if really needed.